src/app/page.tsx

```diff
@@ -1,101 +1,83 @@
-import Image from "next/image";
+"use client";
+import React, { useState } from "react";
+import { Box, Button, Grid2, ThemeProvider, useTheme, Tooltip } from "@mui/material";
+import FileUploaded from "@/classe/File";
+import Dropzone from "@/components/Dropzone";
+import FileList from "@/components/FileList";
+import type { DropzoneState } from "@/types";
 
-export default function Home() {
-  return (
-    <div className="grid grid-rows-[20px_1fr_20px] items-center justify-items-center min-h-screen p-8 pb-20 gap-16 sm:p-20 font-[family-name:var(--font-geist-sans)]">
-      <main className="flex flex-col gap-8 row-start-2 items-center sm:items-start">
-        <Image
-          className="dark:invert"
-          src="https://nextjs.org/icons/next.svg"
-          alt="Next.js logo"
-          width={180}
-          height={38}
-          priority
-        />
-        <ol className="list-inside list-decimal text-sm text-center sm:text-left font-[family-name:var(--font-geist-mono)]">
-          <li className="mb-2">
-            Get started by editing{" "}
-            <code className="bg-black/[.05] dark:bg-white/[.06] px-1 py-0.5 rounded font-semibold">
-              src/app/page.tsx
-            </code>
-            .
-          </li>
-          <li>Save and see your changes instantly.</li>
-        </ol>
+function HomePage() {
+  const theme = useTheme();
+  const [dropzoneState, setDropzoneState] = useState<DropzoneState>({
+    visible: false,
+    image_url: null,
+    fillPercentage: 0,
+  });
+  const [uploadedFiles, setUploadedFiles] = useState<FileUploaded[]>([]);
 
-        <div className="flex gap-4 items-center flex-col sm:flex-row">
-          <a
-            className="rounded-full border border-solid border-transparent transition-colors flex items-center justify-center bg-foreground text-background gap-2 hover:bg-[#383838] dark:hover:bg-[#ccc] text-sm sm:text-base h-10 sm:h-12 px-4 sm:px-5"
-            href="https://vercel.com/new?utm_source=create-next-app&utm_medium=appdir-template-tw&utm_campaign=create-next-app"
-            target="_blank"
-            rel="noopener noreferrer"
+  return (
+    <ThemeProvider theme={theme}>
```

Comment:
Please check if it's necessary to use the ThemeProvider wrapper again here. We already wrapped the components with it in the main layout.tsx.